common/dynamicconfig/constants.go

```diff
@@ -1258,6 +1258,12 @@ const (
 	// Default value: FALSE
 	// Allowed filters: N/A
 	ReplicatorProcessorEnablePriorityTaskProcessor
+	// ReplicatorUpperLatency is indicates the max allowed replication latency between clusters
```

Comment:
nit: also indicate the unit for the number? I guess default 40 means 40 seconds?

Reply:
updated